viaacode/hetarchief-client
# fix(arc-624): Do not require login for content pages (404)

https://meemoo.atlassian.net/browse/ARC-624

Add dummy check for 'existing' content pages

to test, reset your toc date of your profile:
![image](https://user-images.githubusercontent.com/1710840/167125407-1fda7368-b0d5-4169-ad02-a1413dc7d038.png)

You should be able to access the content pages (404) without login:
![image](https://user-images.githubusercontent.com/1710840/167125458-6af50b81-24ff-4320-99b5-dcc38c540bb5.png)

You should be able to access the content pages with login, without having to accept the tos:
![image](https://user-images.githubusercontent.com/1710840/167125513-543005eb-4322-4e2f-8ed7-f847f599fcc7.png)

When going to the logged in home or a visitor space, you do need to accept the tos first:
![image](https://user-images.githubusercontent.com/1710840/167125558-222445da-bb38-4dfe-a833-7b3a3c82a5c1.png)
